app/controllers/api/base_controller.rb

```diff
@@ -16,8 +16,15 @@ class BaseController < ActionController::Base
     after_action :log_response_body
 
     rescue_from StandardError do |error|
-      Foreman::Logging.exception("Action failed", error)
-      render_error 'standard_error', :status => :internal_server_error, :locals => { :exception => error }
+      case error
+        when Net::LDAP::Error
+          message = _("Error during LDAP connection :  #{error}")
```

Comment:
since we caught this here, we have no information to which LDAP server we were trying to connect, maybe rescue this exception earlier where the ldap connection is estabilished and render a new custom exception there with message like `_("Error connection to %{url} LDAP server during authentication: %{error}" % {:url => ..., :error => ...}`